packages/fx-core/tests/plugins/solution/testProject/infra/azure/parameters/parameter.template.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
delete the test files after test case finish

Reply:
This is the input test file.

Reply:
In unit test, we may mock the content (as well as other bicep files). We can resolve this next week.

Reply:
This is the mocked file. I read it from file instead of inline code.